apps/modernization-ui/src/page/usePage.tsx

```diff
@@ -23,6 +27,8 @@ const pageReducer = (state: Page, action: Action) => {
             return { ...state, status: Status.Requested, current: action.page };
         case 'reload':
             return { ...state, status: Status.Requested };
+        case 'set-page-size':
+            return { ...state, pageSize: action.pageSize, page: 1, status: Status.Requested };
```

Comment:
should be `current: 1` instead of `page: 1`.  The state stores the current page number in the `current` property.  The `go-to` action has the requested page number in the `page` property so that the action `{ type: 'go-to', page: 5 }` can be read as `go to page 5`

Reply:
🏆